dk-installer.py

```diff
@@ -607,7 +631,9 @@ def execute(self, args):
 
 
 class Installer:
-    def __init__(self):
+    def __init__(self, arguments=None):
+        if arguments is not None:
+            sys.argv[1:] = arguments
```

Comment:
I think it's a bad idea to update the contents of `sys.argv`. The arg parser has a way to set default arguments. I think this has to be used instead